src/index.html

```diff
@@ -2,12 +2,40 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" 
+      content="width=device-width, initial-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/styles.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
   </head>
-  <body>
+  <body class="body">
     <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <img class="product-card__image"
+        src="images/imac.jpeg" alt="imac.main">
```

Comment:
Don't forget about code style
```suggestion
      <img
        class="product-card__image"
        src="images/imac.jpeg"
        alt="imac image"
      >
```